pkg/plugin/interfaces/backup_item_action.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
I know we don't do this a lot, but since `pkg/plugin/interfaces` is meant to be an external-facing package -- can you add package-level godoc?